powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/v3/CallbackUrlBehavior.java

```diff
@@ -91,9 +87,9 @@ public void setConfiguration(PowerAuthServiceConfiguration configuration) {
     }
 
     /**
-     * Configure WebClient HTTP connection parameters.
+     * Configure Rest client HTTP connection parameters.
      */
-    private void configureWebClient() {
+    private void configureRestClient() {
```

Comment:
Do we really need this method? It is only used once, right before initializing the client, and the only purpose is to store values from configuration to local properties, that I also think we no longer need.

Reply:
True, we can set the configuration directly.